src/foam/u2/view/BadgeView.js

```diff
@@ -0,0 +1,85 @@
+/**
```

Comment:
Wrong copyright header.
Use:
/**
 * @license
 * Copyright 2021 The FOAM Authors. All Rights Reserved.
 * http://www.apache.org/licenses/LICENSE-2.0
 */